css-inline/tests/test_inlining.rs

```diff
@@ -940,6 +945,27 @@ fn keep_link_tags() {
     );
 }
 
+#[test]
+fn keep_at_rules() {
+    let inliner = CSSInliner::options().keep_at_rules(true).build();
+    let html = r#"
+<html>
+<head>
+<style>
+h1 { color: blue; }
+@media (max-width: 600px) { h1 { font-size: 18px; } }
+p { margin: 10px; }
+</style>
+</head>
+<body>
+<h1>Hello</h1><p>World</p>
+</body>
+</html>"#;
+    let inlined = inliner.inline(html).unwrap();
+    let expected = "<html><head><style>@media (max-width: 600px)  { h1 { font-size: 18px; } }</style>\n\n</head>\n<body>\n<h1 style=\"color: blue;\">Hello</h1><p style=\"margin: 10px;\">World</p>\n\n</body></html>";
```

Comment:
I see there is a double space - `(max-width: 600px)  {`. I guess we can keep what is in the original string without adding extra space - it should be a bit cleaner + a bit faster